src/components/structures/auth/CompleteSecurity.tsx

```diff
@@ -75,12 +75,9 @@ export default class CompleteSecurity extends React.Component<IProps, IState> {
         } else if (phase === Phase.ConfirmSkip) {
             icon = <span className="mx_CompleteSecurity_headerIcon mx_E2EIcon_warning" />;
             title = _t("common|are_you_sure");
-        } else if (phase === Phase.Busy) {
+        } else if (phase === Phase.Busy || phase === Phase.ConfirmReset) {
             icon = <span className="mx_CompleteSecurity_headerIcon mx_E2EIcon_warning" />;
             title = _t("encryption|verification|after_new_login|verify_this_device");
-        } else if (phase === Phase.ConfirmReset) {
-            icon = <span className="mx_CompleteSecurity_headerIcon mx_E2EIcon_warning" />;
-            title = _t("encryption|verification|after_new_login|reset_confirmation");
```

Comment:
I'm a bit confused about what's going on here. Don't we still need a confirmation step? why do we show "verify this device" (?) when we're in the "ConfirmReset" phase?

Reply:
I've re-added this branch of the `if` statement